www_src/pages/page/page.less

```diff
@@ -38,30 +38,44 @@
     }
   }
 
-  .overlay {
+  .overlay-enter {
     position: fixed;
     top: 0;
     left: 0;
     height: 100%;
     width: 100%;
-    z-index: 998;
+    /* Must be above everything to prevent people clicking on elements below it*/
+    /* this is the maximum z-index. */
+    z-index: 2147483647;
     background: #1A2237;
     opacity: 0;
     visibility: hidden;
-    transition: all 0.1s ease-in-out;
-    &.active {
+    transition: all 1s ease-in-out;
+
+    &.overlay-enter-active {
       visibility: visible;
       opacity: 0.7;
     }
   }
 
+  /* Necessary to force-remove ReactCSSTransitionGroup elements in unminified React: */
+  .overlay-leave {
+    opacity: 1;
+    transition: opacity .5s ease-in;
```

Comment:
Let's make this transition:

`transition: all 0.3s cubic-bezier(0.79, 0.14, 0.15, 0.86) 0s;`